homeassistant/components/eps/const.py

```diff
@@ -0,0 +1,22 @@
+"""Constants for the iAlarm integration."""
+from pyepsalarm import EPS
+
+from homeassistant.const import (
+    STATE_ALARM_ARMED_AWAY,
+    STATE_ALARM_ARMED_NIGHT,
+    STATE_ALARM_DISARMED,
+    STATE_ALARM_TRIGGERED,
+)
+
+DATA_COORDINATOR = "eps"
+
+DEFAULT_PORT = 18034
+
+DOMAIN = "eps"
+
+EPS_TO_HASS = {
+    EPS.ARMED_AWAY: STATE_ALARM_ARMED_AWAY,
+    EPS.ARMED_NIGHT: STATE_ALARM_ARMED_NIGHT,
+    EPS.DISARMED: STATE_ALARM_DISARMED,
+    EPS.TRIGGERED: STATE_ALARM_TRIGGERED,
+}
```

Comment:
Move single use constants to where they are used

Reply:
Thanks !